Package.swift

```diff
@@ -541,12 +541,12 @@ if ProcessInfo.processInfo.environment["SWIFTPM_LLBUILD_FWK"] == nil {
 
 if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
     package.dependencies += [
-        .package(url: "https://github.com/apple/swift-tools-support-core.git", .branch(relatedDependenciesBranch)),
+        .package(url: "https://github.com/pcbeard/swift-tools-support-core.git", .branch("AvoidWaiting")),
```

Comment:
This will be changed after your TSC PR has been merged, I assume?

Reply:
Yes. I'm removing this commit now.